src/porepy/fracs/msh_2_grid.py

```diff
@@ -467,3 +492,93 @@ def create_embedded_line_grid(
     # Add mapping to global point numbers.
     g.global_point_ind = glob_id[sort_ind]
     return g
+
+
+def tag_grid(
+    sd: pp.Grid,
+    phys_names: dict[int, str],
+    cell_info: dict[str, np.ndarray],
+) -> pp.Grid:
+    """Translate gmsh tags to PorePy tags.
+
+    This functions loops through all geometric elements of a gmsh grid and translates
+    gmsh tags, describing which physical entity each element belongs to, to tags in the
```

Comment:
Suggestion:
gmsh tags to tags in the PorePy grid. The tags describe which physical entity each element belongs to.